client/core/core.go

```diff
@@ -5203,6 +5241,82 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		tracker.changeLocked = true
 	}
 
+	if form.Queue {
+		tempID, err := order.IDFromBytes(encode.RandomBytes(order.OrderIDSize))
+		if err != nil {
+			return nil, fmt.Errorf("unable to generate temporary order ID: %w", err)
+		}
+		tracker.metaData.TempID = tempID
```

Comment:
Just `rand.Read(tracker.metaData.TempID[:])` (using `crypto/rand`).  No need to use these helpers or another variable.